# HG changeset patch
# User Kai Engert <kaie@kuix.de>
# Date 1673640495 -3600
#      Mon Dec 05 13:43:16 2022 +0100
# Node ID e0505ed4940741f3d11bce8d4d226dc7f1ab21f2
# Parent  5f5174285bacab23d5c81a5d5f389e4e662d52bf
Bug 1791130 - Speed up s/mime recipient certificate check. r=mkmelin

  Only hunk left is https://hg.mozilla.org/comm-central/rev/756767a50a8a from
  https://bugzilla.mozilla.org/show_bug.cgi?id=1808295

diff --git a/mail/extensions/am-e2e/prefs/e2e-prefs.js b/mail/extensions/am-e2e/prefs/e2e-prefs.js
--- a/mail/extensions/am-e2e/prefs/e2e-prefs.js
+++ b/mail/extensions/am-e2e/prefs/e2e-prefs.js
@@ -44,17 +44,17 @@ pref("mail.openpgp.alias_rules_file", ""
 pref("mail.openpgp.key_assistant.enable", true);
 
 // If false, disable the reminder in composer, whether email could be
 // sent with OpenPGP encryption (without further user actions/decisions).
 pref("mail.openpgp.remind_encryption_possible", true);
 
 // If false, disable the reminder in composer, whether email could be
 // sent with S/MIME encryption (without further user actions/decisions).
-pref("mail.smime.remind_encryption_possible", false);
+pref("mail.smime.remind_encryption_possible", true);
 
 // When sending, encrypt to this additional key. Not available in release channel builds.
 pref("mail.openpgp.debug.extra_encryption_key", "");
 
 // Ignore missing MDC.
 pref("mail.openpgp.ignore_missing_mdc", false);
 
 // Hide prefs and menu entries from non-advanced users
